tox.ini

```diff
@@ -27,7 +27,11 @@ commands =
   /bin/bash -c 'grep -iFInrq "UPDATE_FIXTURES = True" ./tests && exit 1 || exit 0'
   /bin/bash -c 'podman pull -q {[base]default_ee} || docker pull -q {[base]default_ee}'
   /bin/bash -c 'podman pull -q {[base]small_test_ee} || docker pull -q {[base]small_test_ee}'
-  /bin/bash -c 'py.test -vvvv -n `python -c "import multiprocessing; import os; print(int(os.environ.get(\"XDIST_CPU\") or multiprocessing.cpu_count()))"` --dist=loadfile --maxfail=15 --durations=100 --cov ansible_navigator --cov share --cov-report term-missing --cov-branch --cov-report=xml:{toxworkdir}/coverage-{envname}.xml --showlocals {posargs}'
+  # if you want to limit multiprocessing locally, define PYTEST_ADDOPTS="-n0 --dist no"
```

Comment:
This suggestion is lying — it will not work unless posargs are _also_ overridden according to the how the pytest args precedence works.

Reply:
@webknjaz intended to suggest......

This suggestion might be incorrect — it will not work unless posargs are also overridden according to the how the pytest args precedence works

Reply:
I updated the change to ensure that `PYTEST_ADDOPTS` does not get shadowed by our default arguments from tox.ini.

As we do not use any hardcoded pytest args inside tox.ini, both of these calling methods have the same effect:
```
PYTEST_ADDOPTS="-n0 --dist no" tox -e py
tox -e py -- -n0 --dist no
```